libcontainer/standard_init_linux.go

```diff
@@ -150,6 +152,18 @@ func (l *linuxStandardInit) Init() error {
 			return err
 		}
 	}
-
-	return system.Execv(l.config.Args[0], l.config.Args[0:], os.Environ())
+	// check for the arg before waiting to make sure it exists and it is returned
+	// as a create time error
+	name, err := exec.LookPath(l.config.Args[0])
```

Comment:
On Thu, May 26, 2016 at 01:20:02PM -0700, Michael Crosby wrote:
> > +	// check for the arg before waiting to make sure it exists and it is returned
> > +	// as a create time error
> > +	name, err := exec.LookPath(l.config.Args[0])
>
> At this time it is required and if we want to change that its a
> different discussion

Required as in “that's what runC will do right now” or required as in
“opencontainers/runtime-spec#384 needs to be updated to list this
requirement”.  The distinction there is mostly in how compliance is
tested in ocitools.